12-project-maintenance.Rmd

```diff
@@ -8,3 +8,121 @@ ottrpal::set_knitr_image_path()
 ```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Major point!! example image"}
 ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g21a84b32106_0_58")
 ```
+
+## Understand best practices for maintaining projects
+
+Many R projects are not really finished, they are either maintained long term or they deprecate. Many folks have many R projects over time which means keeping all your R projects maintained could easily become a time consuming endeavor!
+
+In order to keep maintenance a priority without it using up all your time, we have a few tips to discuss. These tips will help you ease the burden of maintenance and lead to a healthier R project long term.
+
+Building some of the infrastructure we will describe will take a bit more time upfront, but will save you time long term.
```

Comment:
```suggestion
Building some of the infrastructure we will describe will take a bit more time upfront, but will save you time overall.
```

Reply:
used "long term" in the last few sentences

Reply:
also changing to ":" and following - so that the list of tips fits with it a bit more smoothly